nexus/types/src/deployment.rs

```diff
@@ -869,7 +895,6 @@ pub enum BlueprintDatasetFilter {
     JsonSchema,
     Deserialize,
     Serialize,
-    EnumIter,
     Diffable,
 )]
 #[serde(rename_all = "snake_case")]
```

Comment:
I think this needs a 

```suggestion
#[serde(tag = "kind", rename_all = "snake_case")]
```

to fix the OpenAPI test failure complaint.

Reply:
Fixed in [2531105](https://github.com/oxidecomputer/omicron/pull/7286/commits/2531105c56870e196236ad82cc9aad24829c148f).

Thanks!